packages/components/psammead-story-promo/package.json

```diff
@@ -27,7 +27,8 @@
     "@bbc/psammead-test-helpers": "^0.3.3",
     "@bbc/psammead-timestamp": "^0.3.2",
     "react": "^16.8.6",
-    "styled-components": "^4.1.3"
+    "styled-components": "^4.1.3",
+    "@testing-library/react": "^8.0.1"
```

Comment:
Can you install this at the top level of psammead, would be good for every package to have it :) (and as tests are run from top level its fine :) )